atomic_reactor/tasks/binary_container_build.py

```diff
@@ -263,20 +266,21 @@ def build_container(
         # passing stdout=PIPE guarantees that stdout is not None, but the type hints for the
         #   subprocess module do not express that (TL;DR - this is just for type checkers)
         assert build_process.stdout is not None
+        assert build_process.stderr is not None
 
-        last_line = None
+        error_lines = []
 
-        while True:
-            if line := build_process.stdout.readline():
-                yield line
-                # keep the last non-empty, non-whitespace line for an eventual error message
-                last_line = line.rstrip() or last_line
+        for line in build_process.stdout:
+            yield False, line
+        for line in build_process.stderr:
+            if line.rstrip():
+                error_lines.append(line)
+            yield True, line
```

Comment:
does this work as expected, it doesn't mean that everyhing in STDERR is error

Reply:
changed to just take last line as error